day_two_guide/topics/verifying_mtu.adoc

```diff
@@ -0,0 +1,141 @@
+////
+Verifying correct Maximum Transmission Unit (MTU) size
+
+Module included in the following assemblies:
+
+* day_two_guide/environment_health_checks.adoc
+////
+
+Verifying the MTU prevents a possible networking misconfiguration that can
+masquerade as an SSL certificate issue.
+
+When a packet is larger than the MTU size that is transmitted over HTTP, the
+physical network router is able to break the packet into multiple packets to
+transmit the data. However, when a packet is larger than the MTU size is that
+transmitted over HTTPS, the router is forced to drop the packet.
+
+Installation produces certificates that
+provide secure connections to multiple components that include:
+
+* master hosts
+* node hosts
+* infrastructure nodes
+* registry
+* router
+
+These certificates can be found within the `/etc/origin/master` directory for
+the master nodes and `/etc/origin/node` directory for the infra and app nodes.
+
+After installation, you can verify connectivity to the
+`OPENSHIFT_DEFAULT_REGISTRY` using the process outlined in the
```

Comment:
'OPENSHIFT_DEFAULT_REGISTRY' is deprecated at ocp 3.9.  REGISTRY_OPENSHIFT_SERVER_ADDR replaced it.

Reply:
This was written before 3.7 was released. I'm not sure about adding 3.9 stuff as it has been not released yet.